examples/gym/solve_hallway.py

```diff
@@ -19,60 +21,67 @@
 # NOTE: These class definitions need to stay outside of construct_problem
 # or you will error out on not being able to pickle/serialize them.
 
-class CartPoleEnv(AlphaZeroGymEnv):
+class HallwayAlphaZeroEnv(AlphaZeroGymEnv):
     """Lightweight wrapper around the gym env that makes the user implement
     the get_obs method."""
 
-    def __init__(self, **kwargs):
-        super().__init__(gym.envs.make("CartPole-v0"), **kwargs)
-    
+    def __init__(self, size=16, max_steps=16):
+        env = HallwayEnv(size=size, max_steps=max_steps)
+        super().__init__(env)
+
     def get_obs(self) -> np.ndarray:
-        return np.array(self.state)
+        return self.env.get_obs()
```

Comment:
`AlphaZeroGymEnv` seems to want `get_env_obs` as the abstract method?

Reply:
`get_env_obs` is out of date, renamed to `get_obs`